cmd/snap-seccomp/main.go

```diff
@@ -122,6 +122,9 @@ package main
 //#define SCMP_ARCH_S390X ARCH_BAD
 //#endif
 //
+//#ifndef SECCOMP_RET_LOG
+//#define SECCOMP_RET_LOG 0x7ffc0000U
```

Comment:
I looked up the value to this patch so +1
 https://www.redhat.com/archives/linux-audit/2017-February/msg00005.html